readme.md

```
@@ -5,40 +5,40 @@
 ### Requirements
 
 1. Node (check `.nvmrc` for version)
-2. [pnpm](https://pnpm.io/)
+2. [Bun](https://bun.sh/)
 
 ### Install dependencies
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)